osquery/events/linux/inotify.cpp

```diff
@@ -210,7 +231,7 @@ Status INotifyEventPublisher::run() {
     p += (sizeof(struct inotify_event)) + event->len;
   }
 
-  pauseMilli(kINotifyMLatency);
+  //pauseMilli(kINotifyMLatency);
```

Comment:
The intention behind this change was, in the case of overflow we should read as fast as possible because there was already an outer delay loop -
  while (!publisher->isEnding()) {
    // Can optionally implement a global cooloff latency here.
    status = publisher->run();
    if (!status.ok()) {
      break;
    }
    publisher->restart_count_++;
    // This is a 'default' cool-off implemented in InterruptableRunnable.
    // If a publisher fails to perform some sort of interruption point, this
    // prevents the thread from thrashing through exiting checks.
    publisher->pauseMilli(200);
  }
 
earlier I thought of sth like -
if(!overflow) { 
  pauseMilli(kINotifyMLatency);
}